server/connectors/api-connector-jdbc/src/main/scala/com/prisma/api/connector/jdbc/impl/JdbcDatabaseMutactionExecutor.scala

```diff
@@ -18,9 +19,34 @@ case class JdbcDatabaseMutactionExecutor(
     extends DatabaseMutactionExecutor {
   import slickDatabase.profile.api._
 
+  private def runAttached[T](project: Project, query: DBIO[T]) = {
```

Comment:
Can you move that method further down? We should aim at being able to read from top to bottom.